MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1851,6 +1879,22 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	_properties.deviceType = isIntegrated ? VK_PHYSICAL_DEVICE_TYPE_INTEGRATED_GPU : VK_PHYSICAL_DEVICE_TYPE_DISCRETE_GPU;
 	strlcpy(_properties.deviceName, _mtlDevice.name.UTF8String, VK_MAX_PHYSICAL_DEVICE_NAME_SIZE);
 
+	if (supportsMTLGPUFamily(Apple5)) {
+		// This is an Apple GPU. It won't have a 'device-id' property, so fill it in
+		// like on iOS/tvOS.
+		_properties.vendorID = 0x106b;	// Apple's PCI ID
+#if (defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 101600)
+		if (supportsMTLGPUFamily(Apple6)) {
```

Comment:
There are several other places in `MVKDevice.mm` that need to be updated to support `Apple6`, both in `MVK_MACOS` and `MVK_IOS` conditional areas. For example, the iOS version of this code section...plus search for `Apple5` for a couple of others.

Curiously, Apple [makes reference](https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf) to `Apple7` (A14) as well...but it doesn't look to be supported in the SDK in Xcode 12.2b3 yet.

Reply:
> Curiously, Apple [makes reference](https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf) to `Apple7` (A14) as well...but it doesn't look to be supported in the SDK in Xcode 12.2b3 yet.

Not the macOS SDK, or the tvOS SDK for that matter, but it is present in the iOS SDK.